Components/ListSearchResults/Partials/Results/index.twig

```diff
@@ -6,14 +6,14 @@
           <img src="{{ post.thumbnail.src }}" data-object-fit="cover" data-object-position="center center">
         </div>
       {% endif %}
-      <div class="content">
-        <a class="postLink" href="{{ post.link }}">
-          <h3 class="postTitle">{{ post.title }}</h3>
-        </a>
+      <div class="post">
+        <a class="post-title" href="{{ post.link }}">{{ post.title }}</a>
         {% if post.preview|length > 10 %}
-          <p class="excerpt">{{ post.preview.length(40) }}</p>
+          <p class="post-excerpt">{{ post.preview.length(40) }}</p>
```

Comment:
You could extend it to `{{ post.preview.length(40).read_more(false) }}` to not have a read more link and then add the read more link manually via `<a href="{{ post.link }}">{{ readMore }}</a>`. That way, I think you won't need the extra post-footer element.